fix!: Authz request parameter wallet_issuer renamed to wallet_id (#142)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

:  Ability of the Holder to prove legitimate possession of a Verifiable Credential by demonstrating a certain biometric trait, such as fingerprint or face. One example of a Verifiable Credential with Biometrics-based Holder Binding is a mobile driving license [@ISO.18013-5], which contains a portrait of the holder.

Wallet:

> :  An entity used by the Holder to request, receive, store, present, and manage Verifiable Credentials and key material. There is no single deployment model of a Wallet: Verifiable Credentials and keys can both be stored and managed locally, or by using a remote self-hosted service, or a remote third-party service.

Reviewer: ```suggestion
:  An entity used by the Holder to request, receive, store, present, and manage Verifiable Credentials and key material. There is no single deployment model of a Wallet: Verifiable Credentials and keys can both be stored and managed locally, or by using a remote self-hosted service, or a remote third-party service. In the context of this specification, during Credential Issuance the Wallet acts as an OAuth 2.0 Client (see [@!RFC6749]) towards the Credential Issuer which acts as the OAuth 2.0 Authorization Server / Resource Server.
```
I am not sure we should remove this sentence entirely, as opposed to making it accurate.

Author: I fully agree your rewording @Sakurann, now the text reflect precisely the context,  the roles and the scope of this specs. Text picked!

> :  An entity used by the Holder to request, receive, store, present, and manage Verifiable Credentials and key material. There is no single deployment model of a Wallet: Verifiable Credentials and keys can both be stored and managed locally, or by using a remote self-hosted service, or a remote third-party service. In the context of this specification, during Credential Issuance the Wallet acts as an OAuth 2.0 Client (see [@!RFC6749]) towards the Credential Issuer which acts as the OAuth 2.0 Authorization Server / Resource Server.

Author: ```suggestion
:  An entity used by the Holder to request, receive, store, present, and manage Verifiable Credentials and key material. There is no single deployment model of a Wallet: Verifiable Credentials and keys can both be stored and managed locally, or by using a remote self-hosted service, or a remote third-party service. In the context of this specification, the Wallet acts as an OAuth 2.0 Client (see [@!RFC6749]) since the Credential Issuer serves as the OAuth 2.0 Resource Server and, in certain implementations, it may also perform the role of the OAuth 2.0 Authorization Server. 
```

Author: @Sakurann WDYT?

* A new authorization details [@!RFC9396] type `openid_credential` is defined to convey the details about the Credentials (including formats and types) the Wallet wants to obtain (see (#authorization-details)).
* New token response error codes `authorization_pending` and `slow_down` are added to allow for deferred authorization of Credential issuance. These error codes are supported for the Pre-Authorized Code grant type.
* Client metadata is used to convey Wallet's metadata. A new metadata parameter `credential_offer_endpoint` is added to allow a Wallet (acting as OAuth 2.0 client) to publish its Credential Offer Endpoint (see (#client-metadata)).

> * Authorization Endpoint: An additional parameter `issuer_state` is added to convey state in the context of processing an issuer-initiated Credential Offer (see (#credential-authz-request)). Additional parameters `wallet_id` and `user_hint` are added to enable the Credential Issuer to request Verifiable Presentations from the calling Wallet during Authorization Request processing.

Author: ```suggestion
* Authorization Endpoint: An additional parameter `issuer_state` is added to convey state in the context of processing an issuer-initiated Credential Offer (see (#credential-authz-request)). In conjunction with `client_id`, the additional parameter `user_hint` enables the Credential Issuer to request Verifiable Presentations from the calling Wallet during the Authorization Request processing.
```
